<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetBrowser</class>
 <widget class="QMainWindow" name="TargetBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADS Target Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="pathLayout"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>&amp;Search:</string>
        </property>
        <property name="buddy">
         <cstring>searchInput</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchInput">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="targetView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Connect_to_recent"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Symbols"/>
    <addaction name="actionExport_Data_Types"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_remote_rou_te"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy_full_name"/>
    <addaction name="action_Read_value"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;New Connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Connect_to_recent">
   <property name="text">
    <string>Connect to &amp;recent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Copy_full_name">
   <property name="text">
    <string>&amp;Copy full name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Read_value">
   <property name="text">
    <string>&amp;Read value</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionCreate_remote_rou_te">
   <property name="text">
    <string>Create remote r&amp;oute</string>
   </property>
  </action>
  <action name="actionExport_Symbols">
   <property name="text">
    <string>Export &amp;Symbols</string>
   </property>
  </action>
  <action name="actionExport_Data_Types">
   <property name="text">
    <string>Export Data-&amp;Types</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
